src/mscorlib/src/System/Environment.cs

```diff
@@ -369,6 +369,9 @@ public static String CurrentDirectory
             [System.Security.SecuritySafeCritical]  // auto-generated
 #endif
             get {
+#if PLATFORM_UNIX
+                return Path.DirectorySeparatorCharAsString; // TODO: anything better to return?
```

Comment:
Hmmm, not sure what is the right answer here. Both seems equally useless. Throwing `PlatformNotSupportedException` on Unix may be most appropriate.